README.md

```diff
@@ -41,18 +41,24 @@
 
 ## Guiding Principles
 
-* **Modern, reliable foundation for everyday bioinformatics analysis—written in Rust.** `sprocket` aims to package together a fairly comprehensive set of tools and for developing bioinformatics tasks and workflows using the [Workflow Description Language](http://openwdl.org/). It is built with modern, multi-core systems in mind and written in Rust.
-* **WDL specification focused.** We aim to implement the various versions of the [OpenWDL specification](https://github.com/openwdl/wdl) to the letter. In other words, `sprocket` aims to be workflow engine independent. In the future, we plan to make `sprocket` extendable for workflow engine teams.
+* Provide a **high-performance** workflow execution engine capable of
+  orchestrating massive bioinformatics workloads (the stated target is 20,000+
```

Comment:
Just thoughts/questions that don't necessarily need to be answered _in_ the document:

Where does the number 20,000 come from? And how is a "job" defined? Asking because I noticed in `cromwell server` that a limiting factor of performance wasn't "number of concurrent workflows" but "number of concurrent tasks". e.g. our `rnaseq-standard` workflow is relatively linear, where QC is highly parallel. Cromwell could handle many more `rnaseq-standard` jobs than it could QC jobs.

Reply:
Good question. To me it means "workflows in progress". So that is, perhaps, even more ambitious than tasks. Note that a workflow that is "pending" still counts in my book (it's really just the practice of 'submit and forget' and you're confident that these will eventually run to completion).

The 20,000 number came from my head. I'd like to pick a concrete number to shoot for, and this seemed like a high number that was also in the realm of achievable. For now, I'd like to stick with that.